soc/riscv/riscv-privilege/nuclei/gd32vf103/soc.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2021 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <soc.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <nuclei_csr.h>
+#include <init.h>
```

Comment:
nit: review include list, some of them don;t seem to be required

Reply:
Only init.h is needed.
Removing others.